src/ray/core_worker/task_manager.cc

```diff
@@ -126,6 +127,7 @@ bool TaskManager::ResubmitTask(const TaskID &task_id, std::vector<ObjectID> *tas
       resubmit = true;
       it->second.SetStatus(rpc::TaskStatus::PENDING_ARGS_AVAIL);
       it->second.MarkRetryOnResubmit();
+      RecordTaskStatusEvent(it->second, rpc::TaskStatus::PENDING_ARGS_AVAIL);
```

Comment:
Could we unify these calls with `SetStatus` (to prevent accidentally forgetting to call RecordTaskStatusEvent when calling SetStatus), here and elsewhere?